WordPress/src/main/java/org/wordpress/android/datasets/CommentTable.java

```diff
@@ -95,7 +95,7 @@ public static void addComment(int localBlogId, final Comment comment) {
         values.put("comment_id",        comment.commentID);
         values.put("author_name",       comment.getAuthorName());
         values.put("author_url",        comment.getAuthorUrl());
-        values.put("comment",           comment.getCommentText());
+        values.put("comment",           SqlUtils.maxSQLiteText(comment.getCommentText()));
```

Comment:
Yes, it's a 2Mb limit per row, but other fields should be relatively small compared to `commentText` - and we have already a margin for error on the `maxSQLiteText` returned.